### ISSUE TITLE
apertium script should be able to use apy

### ISSUE THREAD

**Issue by ftyers** [apertium/apertium#3]
It would be cool if the apertium script could make calls to apy with a flag, like:

```
$ echo "something" | apertium -p eng-cat
```

Would send the request to apy instead of looking locally, alternatively `apertium eng-cat` could first check for a running apy instance and use it if available.

**Comment by unhammer** [comment]
So `apertium eng-cat` (without `-p`) would prefer `localhost:2737` over opening a new pipeline?

**Comment by ftyers** [comment]
Yeah, i'm not sure about the exact behaviour, but it could be something like:

* Collect all installed modes (local, apy)
* One flag to prefer local, default to apy, OR
* One flag to prefer apy, default to local

In any case, the important thing is being able to use it.

**Comment by unhammer** [comment]
> <TinoDidriksen> It'd just be piping to nc, but how would it know that APy is
>               running and has that pair? Doing a listPairs every time sounds
>               heavy, and if APy isn't running then there's a socket timeout to
>               wait for. APy should listen on an Unix socket as well for this
>               to work nicely.  [11:55]

Also, although APy is faster for short requests, it's much slower for large requests, so there should be some logic to decide what to use there.

**Comment by flammie** [comment]
it'd be a nice feature to use the apys available in the internet too. Maybe have env.var. APY_HOST that defaults to localhost:7327.

**Comment by unhammer** [comment]
I understood the `-p` option as meaning "use internets", e.g. defaulting to www.apertium.org (I think some people might find it a bit disconcerting if `/usr/bin/apertium eng-cat` without any options given could send your texts over the net).

I think it'd be nice to have both options out-of-the-box, so after a fresh install of apertium without any deps or configuration you can `apertium -p eng-cat`.

**Comment by b-thebest** [comment]
Hello, Can I start working on this issue, I have experience of woking with python. I want to contribute in open source projects too.

**Comment by TinoDidriksen** [comment]
@b-thebest, the [`apertium` wrapper script](https://github.com/apertium/apertium/blob/master/apertium/apertium-header.sh) is not Python - it's Bash.

**Comment by ftyers** [comment]
@b-thebest yes you could definitely start working on this. You'll need to know Bash as well as Python and the best thing is to join us on IRC.

**Comment by chiru200513** [comment]
is there anyone currently working in it ? Cause i thought of working in this issue
